mungegithub/mungers/submit-queue.go

```diff
@@ -545,6 +547,7 @@ func (sq *SubmitQueue) EachLoop() error {
 // RegisterOptions registers options used by the SubmitQueue.
 func (sq *SubmitQueue) RegisterOptions(opts *options.Options) {
 	opts.RegisterStringSlice(&sq.NonBlockingJobNames, "nonblocking-jenkins-jobs", []string{}, "Comma separated list of jobs that don't block merges, but will have status reported and issues filed.")
+	opts.RegisterStringSlice(&sq.PreSubmitJobNames, "presubmit-jenkins-jobs", []string{}, "Comma separated list of jobs that run before merging code.")
```

Comment:
The munger has no notion of where a presubmit job runs since it merely looks at github status contexts or a GCS bucket. Should we stop adding "jenkins" into these flags?